packages/core/src/state-management/__tests__/repository.test.ts

```diff
@@ -578,18 +578,26 @@ describe('validation', () => {
     await TestUtils.anchorUpdate(ceramic, schema)
     // Create invalid stream
     const ipfs2 = await createIPFS()
+    await swarmConnect(ipfs, ipfs2)
+
     const permissiveCeramic = await createCeramic(ipfs2)
+
+    // Wait for peers to discover they are both subscribed to same topic
+    await TestUtils.delay(500)
+
     const validateSchemaSpy = jest.spyOn(
       (permissiveCeramic._streamHandlers.get('tile') as TileDocumentHandler)._schemaValidator,
       'validateSchema'
     )
     // eslint-disable-next-line @typescript-eslint/no-empty-function
-    validateSchemaSpy.mockImplementation(() => {})
+    validateSchemaSpy.mockImplementation(() => { })
     const invalidDoc = await TileDocument.create(
       permissiveCeramic,
       { stuff: 1 },
       { schema: schema.commitId }
     )
+
```

Comment:
make sure to always run prettier to clean up code formatting!

Reply:
I did, maybe it doesn't catch this change?

Reply:
there are a bunch of small formatting issues throughout this PR, I only pointed out this one though.  Maybe make sure your IDE is set up to run it on save?  You should also be able to run `npm run format` in the js-ceramic root dir to format the whole repo.

Reply:
My IDE is setup to run it on save, however it does change various existing formatting. Maybe there is a missing config my IDE needs to find? 

I tried `npm run format` but the `format` script is not defined in package.json (toplevel).

I ran `npx prettier -w packages/**/src/**/*.ts`  before pushing as it was the closest command I could find in the package.json.

Reply:
Oh yeah, you're right, my bad.  It was the ceramic-integration-tests repo where I saw a "format" script: https://github.com/3box/ceramic-integration-tests/blob/213925d06e546ef7ad6db4ff78215eb84269dabb/package.json#L24

Reply:
> My IDE is setup to run it on save, however it does change various existing formatting.

hmm interesting.  Maybe also double-check what version of prettier you are using?